recipes/lfpy/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set name = 'LFPy' %}
+{% set version = "2.0.2" %}
+{% set build = 0 %}
+
+package:
+  name: {{ name | lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/l/lfpy/{{name}}-{{ version }}.tar.gz
+  sha256: ae146eb7c7723a383bbec438e8f3d9692b8439d873e79391a500a0bdece32bef
+
+build:
+  number: {{ build }}
+  script: "{{PYTHON}} -m pip install . --no-deps -vv"
+  skip: true  # [win]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - python
+    - pip
+    - cython >=0.20
+    - numpy >=1.8
+    - neuron
+  run:
+    - python
+    - setuptools
```

Comment:
Setuptools is listed as a dependency in setup.py. This may not be strictly correct upstream (I didn't find calls to pkg_resources), but in order to satisfy the resulting package metadata from `pip install .`, it's included here.

Reply:
Probably the developers forgot to delete it, you may raise an issue for it! :D

Reply:
Reported upstream: https://github.com/LFPy/LFPy/pull/136

can be removed here once upstream decides to accept the patch and make a release.